deel-3-webpagina/styles/styles.css

```diff
@@ -1 +1,118 @@
-/*Schrijf hier jouw css*/
\ No newline at end of file
+@import url('https://fonts.googleapis.com/css2?family=Golos+Text:wght@400;500;700&display=swap');
+
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
+
+/*------------ Kleuren ------------
+blauw (space cadet) #151E3F
+Oxford Blue #030027
+Beige #F2F3D9
+oranje (Buff) #DC9E82
+Old rose #C16E70
+-------------------------------------*/
+
```

Comment:
Je zou de kleuren ook in een css variable kunnen zetten. Zodat als je een kleur verandert je dat niet op 10 plaatsen moet doen maar centraal op 1

dit doe je als volgt:
```css
::root {
  --color-oranje: #DC9E82;

}

```

vervolgens gebruik je deze ergens anders:

```css

article {
 background-color: var(--color-oranje);
}
```

Reply:
Ah wat goed! Deze ga ik meenemen voor de volgende keer. Thanks voor je feedback!